realsense2_camera/include/base_realsense_node.h

```diff
@@ -70,6 +70,19 @@ namespace realsense2_camera
 
     const std::vector<stream_index_pair> HID_STREAMS = {GYRO, ACCEL, POSE};
 
+    static const rmw_qos_profile_t rmw_qos_profile_latched =
```

Comment:
I think the definition of *rmw_qos_profile_latched* can be in the .cpp file, say before BaseRealSenseNode::BaseRealSenseNode(). It it not related to anything outside the class.

Reply:
The changes are [here](https://github.com/IntelRealSense/realsense-ros/pull/1833/commits/b8aa13aea7246c09eefa77af578384f8c792f7ac)

> It it not related to anything outside the class.

Actually, in this case, shouldn't be better to declare this profile as an attribute of the class `BaseRealSenseNode::BaseRealSenseNode` in the .h file ?

Reply:
Generally, global variables are avoided when possible, especially when there are not related to anything outside some classes (like this one, as you said). So if there is no need to access `rmw_qos_profile_latched` outside from `BaseRealSenseNode`, why should it be declared outside of it, and not be a class member?